observatory-platform/observatory/platform/bigquery.py

```diff
@@ -96,21 +104,12 @@ def bq_table_id_parts(table_id: str) -> Tuple[str, str, str, str, Optional[pendu
     parts = table_id.split(".")
     project_id = parts[0]
     dataset_id = parts[1]
-    table_id = parts[2]
```

Comment:
The type hints for the function should be: `Tuple[str, str, str, Optional[pendulum.Date]]`

Reply:
Ah and the documentation could say: ":return: project_id, dataset_id and table_id and optional shard date."